test/remote/db_tests.js

```diff
@@ -683,15 +683,13 @@ describe('remote db', function() {
         .then(function(passwordForgotToken) {
           return db.forgotPasswordVerified(passwordForgotToken)
             .then(accountResetToken => {
-              assert.ok(accountResetToken.createdAt > passwordForgotToken.createdAt, 'account reset token should be newer than password forgot token')
+              assert.ok(accountResetToken.createdAt >= passwordForgotToken.createdAt, 'account reset token should be equal or newer than password forgot token')
```

Comment:
This fixes https://github.com/mozilla/fxa-auth-server/issues/2534#issuecomment-409617084, where I think the reset token has the same timestamp as forgot token since it gets create right next to each other. I could have added a delay here but opted to just test that they were >= since a they could have same timestamp.